galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/spi/ras/RasTestClass.java

```diff
@@ -0,0 +1,66 @@
+package dev.galasa.framework.spi.ras;
+
+
+public class RasTestClass {
+    
+    public String testClass;
+    public String bundleName;
+
+    public RasTestClass(String testClass, String bundleName) {
+        this.testClass = testClass;
+        this.bundleName = bundleName;
+    }
+
+    public String getTestClass() {
+        return this.testClass;
+    }
+
+    public void setTestClass(String testClass) {
+        this.testClass = testClass;
+    }
+
+    public String getBundleName() {
+        return this.bundleName;
+    }
+
+    public void setBundleName(String bundleName) {
+        this.bundleName = bundleName;
+    }
+
+
+    // public TestClass testClass(String testClass) {
+    //     this.testClass = testClass;
+    //     return this;
+    // }
+
+    // public TestClass bundleName(String bundleName) {
+    //     this.bundleName = bundleName;
+    //     return this;
+    // }
+
+    // @Override
+    // public boolean equals(Object o) {
+    //     if (o == this)
+    //         return true;
+    //     if (!(o instanceof TestClass)) {
+    //         return false;
+    //     }
+    //     TestClass testClass = (TestClass) o;
+    //     return Objects.equals(testClass, testClass.testClass) && Objects.equals(bundleName, testClass.bundleName);
+    // }
+
+    // @Override
+    // public int hashCode() {
+    //     return Objects.hash(testClass, bundleName);
+    // }
+
+    @Override
+    public String toString() {
```

Comment:
toString should be a simple string report not a json structure.   You will use GSON to convert a POJO to a json string.